code/obj/item/device/nuke_summon_remote.dm

```diff
@@ -0,0 +1,26 @@
+/obj/item/remote/nuke_summon_remote
+	name = "Nuclear Bomb Teleporter Remote"
+	icon = 'icons/obj/items/device.dmi'
+	desc = "A single-use teleporter remote that summons the nuclear bomb to the user's current location."
+	icon_state = "bomb_remote"
+	item_state = "electronic"
+	density = 0
+	anchored = 0.0
+	w_class = 2.0
+
+	var/charges = 1
+	var/use_sound = 'sound/machines/chime.ogg'
```

Comment:
shouldn't this be a string not a file, so we use the soundcache?

Reply:
ye, done